bikes/models.py

```diff
@@ -1,38 +1,217 @@
 from django.db import models
 from django.utils import timezone
 from django.urls import reverse #Used to generate URLs by reversing the URL patterns
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.utils.translation import ugettext as _
+from localflavor.us.models import USStateField
+from localflavor.us.models import USZipCodeField
+
+
+class People(models.Model):
```

Comment:
it seems like it might be better to subclass/customize the `User` model here instead, but one of the caveats might be that there is a required `password` which we may not want for the average User. So this model could work for the `User` and `No Waiver` instances, but we wouldn't want to keep the `user` field on line 49 or either of the listeners that are posted after this model `def` lines 79-86, these lines I had to comment out to do 
`python manage.py createsuperuser`  

this is also a test to see if this message comes through in the new slack channel @git

Reply:
appear in the #git channel

Reply:
Yeah the average user should not need a password. If we add volunteer management stuff at some point we will want users to log in to sign up for events. ATM my knowledge of handling users is definitely lacking but have been doing a lot for work so am solid elsewhere.